en/mkdocs.yml

```diff
@@ -268,9 +268,20 @@ nav:
     - 'EMail OTP Configurations': guides/mfa/email-otp-config-advanced.md
     - 'SMS OTP Configurations': guides/mfa/sms-otp-config-advanced.md
     - 'TOTP Configurations': guides/mfa/totp-config-advanced.md
+  - 'Admin Portal':
+      - 'Applications':
+          - admin-portal/applications/overview.md
+          - 'Add New Application':
+            - admin-portal/applications/add-application.md
+            - 'Single Page Application': admin-portal/applications/types/spa.md
+            - 'OIDC Web Application': admin-portal/applications/types/oidc.md
+            - 'SAML Web Application': admin-portal/applications/types/saml.md
```

Comment:
can we keep these pages in the 'applications' folder itself instead of adding them inside a further sub folder called 'types'?